terraso_backend/config/settings.py

```diff
@@ -416,3 +417,14 @@ class JWTProvider(TypedDict):
 MAPBOX_API_URL = config("MAPBOX_API_URL", default="https://api.mapbox.com")
 MAPBOX_USERNAME = config("MAPBOX_USERNAME", default="")
 MAPBOX_ACCESS_TOKEN = config("MAPBOX_ACCESS_TOKEN", default="")
+
+sentry_sdk.init(
+    dsn="https://b32e6c8aee5ecfd88a9b405302792a2f@o4506299756642304.ingest.sentry.io/4506345139601408",  # noqa: E501
```

Comment:
Question: Should this be defined through a environment variable?  I don't think this should be used by anyone using the repo